requirements-docs.txt

```diff
@@ -0,0 +1,13 @@
+crate-docs-theme>=0.5.74
+
+# packages for local dev
+sphinx-autobuild
+
+# the next section should exact copy of the hosted RTD environment (please
+# update if you notice that this is not the case)
+
+alabaster>=0.7,<0.8,!=0.7.5
+setuptools<41
+sphinx==1.7.4
+sphinx-argparse==0.2.5
```

Comment:
`sphinx-argparse` and `.` are only required if the docs use the argparse plugin to document the argument parser.